vmas/scenarios/transport.py

```diff
@@ -444,12 +465,17 @@ def partial_observation(self, agent: Agent):
          # get positions of all entities in this agent's reference frame
         package_obs = []
         out_of_obs_val = -0.0001 # default value used for out-of-observation data in the observation vector
+        
+        # spawn the sensor radius for each agent
```

Comment:
is the sensor always attached to the agents?

Reply:
Yes, this is reset in the observation function for partial observability.